app/pages/BrowseResultsPage/BrowseResultsPage.tsx

```diff
@@ -59,12 +60,13 @@ export const BrowseResultsPage = () => {
 
   // TODO: Handle failure?
   useEffect(() => {
+    clearRefinements();
     dataService
       .get(`/api/categories/${category.id}`)
       .then(({ category: serviceCategory }: { category: ServiceCategory }) => {
         setParentCategory(serviceCategory);
       });
-  }, [category.id]);
+  }, [category.id, clearRefinements]);
```

Comment:
**note:** lint made me add this to the useEffect dependencies but I don't really understand why it needs to be here

Reply:
Because functions are also reactive values, and that variable may not be the reference to the same function on every re-render! 

Is there a potential design here where you could call `clearRefinements` upon a user interaction instead of doing it _reactively_ on a component (re)render? Such as when the `Clear All` button is clicked? This tends to be a _sturdier_ design anyway as it will be obvious when the clearing happens and it will happen closer to the moment the user does a thing.

Reply:
There is! But I'd have to change the DropdownMenu which takes in URLs and would have to take in callback functions instead which felt weird on a dropdown select.. What do you think?